src/app/containers/Article/__snapshots__/index.test.jsx.snap

```diff
@@ -1,66 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ArticleContainer Component no data should render correctly 1`] = `null`;
-
-exports[`ArticleContainer Component should render Persian article correctly 1`] = `
+exports[`ArticleContainer Component 200 status code should render correctly 1`] = `
 <React.Fragment>
   <GlobalStyleComponent />
   <ServiceContextProvider
-    service="persian"
+    service="news"
```

Comment:
We have seemingly lost the persian article render as part of these changes, before it add a news and a persian render now it only has news. Could you re-add please

Reply:
I've just notice that `ArticleMain` uses the persian data. However, I still think it' beneficial to have Persian data at this level as it still includes the metaData render, whereas the `ArticleMain` snapshots do not